### ISSUE TITLE
Could I use pi-gen to make an x86 Linux distribution?

### ISSUE THREAD

**Issue by GeoffLedak** [RPi-Distro/pi-gen#647]
Hi. I generally don't use GitHub issues to ask questions, but this seemed like the best place to ask. I put together a custom image for the Raspberry Pi using pi-gen. I liked using pi-gen and I have a decent understanding of it at this point. Would it be possible to use pi-gen to create an image for x86/x64 machines? Or would a different tool be a better option..

Thanks!

**Comment by XECDesign** [comment]
Yeah, we use it to generate the root filesystem for our x86 images.

I think it mostly depends on what you're after. If you want to stick close to Debian, it might make sense to use their tools.

https://wiki.debian.org/SystemBuildTools#General_tools

Even after you have a suitable rootfs, making a live cd from it still needs Debian's tools and integrating it all together is a bit of a pain.
